### PR TITLE
Added custom NodePath.prototype.toString method as debug utility

### PR DESCRIPTION
**Pull request by Andarist**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | 
| Patch: Bug Fix?          | 
| Major: Breaking Change?  | 
| Minor: New Feature?      | yes
| Tests Added + Pass?      |
| Documentation PR         | 
| Any Dependency Changes?  |
| License                  | MIT

Just a debug utility, I've found the pattern useful when debugging. Continuing inspecting ASTs is quite mind-bending over course of longer debugs.

### CODE REVIEW

**Comment by nicolo-ribaudo** [comment]
It looks like ci failures are not related?

**Comment by Andarist** [comment]
They shouldnt be related I think, I've restarted CI to see what happens.

**Comment by Andarist** [comment]
CI is failing because of new `browserslist` version with a bug (it's getting fixed right now). How did it get installed though? We have a lock file after all... 😢

**Comment by nicolo-ribaudo** [comment]
> We have a lock file after all... 

I think that lockfiles are ignored by default, unless you specify a flag I don't remember right now

File: packages/babel-traverse/src/path/index.js [hunk 1]
```diff
@@ -7,6 +7,7 @@ import traverse from "../index";
 import Scope from "../scope";
 import * as t from "@babel/types";
 import { path as pathCache } from "../cache";
+import generator from "@babel/generator";
```
Comment (loganfsmyth) [inline packages/babel-traverse/src/path/index.js]: This is just a devDependency at the moment, so this is going to throw when installed in production.

Comment (hzoo) [reply packages/babel-traverse/src/path/index.js]: Also is this going to be a circular dep if generator uses traverse?

File: packages/babel-traverse/src/path/index.js [hunk 2]
```diff
@@ -146,6 +147,10 @@ export default class NodePath {
     if (!debug.enabled) return;
     debug(`${this.getPathLocation()} ${this.type}: ${message}`);
   }
+
+  toString() {
+    return generator(this.node).code;
```
Comment (hzoo) [inline packages/babel-traverse/src/path/index.js]: Or could we somehow make this change in babel-core which does have it as a dep?

File: packages/babel-traverse/src/path/index.js (hunk 1 above)
Comment (loganfsmyth) [reply packages/babel-traverse/src/path/index.js]: It doesn't, so I think we're fine for the dependency-cycle case. I'm happy with just adding it as a dependency for now.

Comment (Andarist) [reply packages/babel-traverse/src/path/index.js]: oh, my bad, ive checked `package.json` too quickly, didnt notice its just a devDep - fix is on the way

File: packages/babel-traverse/src/path/index.js (hunk 2 above)
Comment (Andarist) [reply packages/babel-traverse/src/path/index.js]: any ideas on how that would look like? at the moment it's as easy as doing:
```js
CallExpression(path) {
  console.log(path + '')
}
```

File: packages/babel-traverse/src/path/index.js (hunk 1 above)
Comment (loganfsmyth) [reply packages/babel-traverse/src/path/index.js]: No problem.

**Comment by jridgewell** [comment]
<h1>THIS IS THE BEST THING EVER</h1>

Why did we never think of this before? How many hours have I wasted stepping into object literals to try an figure out what AST tree I was mutating? ❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️❤️

**Pull request merged**